cregistry/chain_info.go

```diff
@@ -83,7 +83,8 @@ type ChainInfo struct {
 			Provider string `json:"provider"`
 		} `json:"rest"`
 	} `json:"apis"`
-	ExtraCodecs []string `json:"extra_codecs"`
+	ExtraCodecs  []string `json:"extra_codecs"`
+	MaxGasAmount uint64   `json:"max-gas-amount"`
```

Comment:
to be consistent with [chain-registry](https://github.com/cosmos/chain-registry)
```suggestion
	MaxGasAmount uint64   `json:"max_gas_amount"`
```